### DISCUSSION TITLE
error: its MIME type, “text/html”, is not “text/css”.

### DISCUSSION THREAD

**Discussion by arunkumar413** [vercel/community#1031] [Help] [answered]
Hi All, 

I'm trying to publish this ( https://github.com/arunkumar413/serene-UI ) react app to vercel. It tried to deploy and auto assigned domains to preview but the app isn't accessible on the assigned domains. I see these error messages in the console.

```
The stylesheet https://serene-ui.vercel.app/serene-ui/normalize.css was not loaded because its MIME type, “text/html”, is not “text/css”. serene-ui.vercel.app
The stylesheet https://serene-ui.vercel.app/serene-ui/static/css/main.5c3a6a7d.css was not loaded because its MIME type, “text/html”, is not “text/css”. serene-ui.vercel.app
Uncaught SyntaxError: expected expression, got '<'
main.494fcb2c.js:1

```

Any help in this regard will be highly appreciated.

Thanks,
Arun

**Comment by swarnava** [comment]
Hi,

I just deployed your app https://serene-ui.vercel-support.app/ and don't see much difference. How are you reproducing this?

**Reply by arunkumar413** [reply]
@swarnava I just removed the homepage property in package.json and built the app again. This time I see this message in the console.

```
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content.
```

**Comment by arunkumar413** [comment]
This link https://serene-ui.vercel-support.app/  doesn't render the app

**Comment by swarnava** [comment, marked as answer]
Should we try replicating using https://github.com/arunkumar413/serene-UI?

**Reply by arunkumar413** [reply]
Seems like the issue is caused due to the home page property  in package.json file. I seems to have messed up the paths. After removing the home page property and fixing some paths in the link tags, it's working as expected on this link https://serene-ui.vercel.app/serene-ui/docs
